setup.py

```diff
@@ -32,11 +32,9 @@ def read(*parts):
     packages=find_packages(),
     test_suite='eve_sqlalchemy.tests',
     install_requires=[
-        'Eve>=0.6,<0.7',
-        'Flask-SQLAlchemy>=1.0,<2.999',
-        'SQLAlchemy>=1.1',
-        # keep until pip properly resolves dependencies:
-        'Flask<0.11',
+        'Eve<0.8',
+        'Flask-SQLAlchemy',
+        'SQLAlchemy',
```

Comment:
I'm not sure what the best practice is here: Pinning to the current major version, assuming some kind of SemVer and expected breaking changes between major versions? Or add the version limits as soon as it is known that there are restrictions... (as done here). I'm fine to follow your expertise here.

Reply:
After all these years, I am still unsure about this one. 

Pinning is supposed to guarantee stability, but then (some) people will invariably go and update the dependencies on their own (usually after a pip-review or something), and then come here and open tickets if/when breakage occurs. 

On the other hand, not pinning makes sure compatibility issues surface as soon as possible, so we can fix them rapidly.

Given the slow release cycle we have with this project, it is probably better to pin the dependencies, opting for stability.